crates/project/src/dap_store.rs

```diff
@@ -1569,6 +1576,20 @@ impl DapStore {
         })
     }
 
+    async fn handle_get_debugger_sessions(
+        this: Model<Self>,
+        _envelope: TypedEnvelope<proto::GetDebuggerSessions>,
+        mut cx: AsyncAppContext,
+    ) -> Result<()> {
+        this.update(&mut cx, |dap_store, cx| {
+            if let Some((_, _)) = dap_store.downstream_client {
+                cx.emit(DapStoreEvent::SendDebuggerSessions(ProjectId(
+                    _envelope.payload.project_id,
```

Comment:
What does this event `SendDebuggerSessions` do? No one is acting on it.

Reply:
Debug panel is supposed to act on it to send active sessions. Im planning on adding a client handler directly to the debug panel to clean this up